jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/JibExtensionTest.java

```diff
@@ -85,6 +86,21 @@ public void testTo() {
     Assert.assertEquals("some cred helper", testJibExtension.getTo().getCredHelper());
     Assert.assertEquals("some username", testJibExtension.getTo().getAuth().getUsername());
     Assert.assertEquals("some password", testJibExtension.getTo().getAuth().getPassword());
+
+    // Testing scenario where jib.to.tags is set to a collection containing a null value.
+    HashSet<String> tags = new HashSet<String>();
+    tags.add(null);
```

Comment:
if we fail on empty, we should probably add a test for empty as well

Reply:
Also the case of getting tags from the system property.

Reply:
good catch!